src/library/scala/collection/mutable/ListBuffer.scala

```diff
@@ -274,7 +288,7 @@ class ListBuffer[A]
   }
 
   def mapInPlace(f: A => A): this.type = {
-    ensureUnaliased()
+    tracker.addMutation()
```

Comment:
I don't follow why this should be `addMutation` instead of `ensureUnaliased`.

Reply:
Oops, yes I do.  Because you don't alter the old list anyway--it's the same logic as used to ensure unaliased.  Never mind!

Reply:
because it builds a completely fresh instance and then just copies references, so it doesn't matter if it's aliased. although it would be useful to set it to `aliased = false` I guess...